Language-Version-History.md

```diff
@@ -0,0 +1,13 @@
+Features Added in VB Language Versions
+====================
```

Comment:
Let's add 

*NOTE:* See how to [specify the language version for your project here](https://docs.microsoft.com/en-us/dotnet/visual-basic/language-reference/configure-language-version)